### PR TITLE
Verified icons

### PR DESCRIPTION
**Pull request by rossmoody**
Addresses https://github.com/brave/brave-browser/issues/4167
Addresses https://github.com/brave/brave-browser/issues/4162
Addresses: https://github.com/brave/brave-browser/issues/4043

## Screens
Screen Shot 2019-04-18 at 11 55 07 PM
Screen Shot 2019-04-18 at 11 54 41 PM
Screen Shot 2019-04-18 at 11 54 28 PM
Screen Shot 2019-04-18 at 11 54 23 PM

### CODE REVIEW

**Comment by NejcZdovc** [comment]
Please fix travis

File: stories/assets/locale.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const locale: Record<string, string> = {
   activityNote: 'To protect your privacy, this Brave Rewards statement is not saved, recorded or logged anywhere other than on your device (this computer). It cannot be retrieved from Brave in the event of data loss on your device.',
   addFunds: 'add funds',
   addFundsFAQ: 'the FAQ',
-  addFundsNote: 'Reminder: The Brave Wallet is unidirectional and BAT flows to publisher sites. For more information about Brave Payments, please visit',
+  addFundsNote: 'Reminder: The Brave Wallet is unidirectional and BAT flows to creators sites. For more information about Brave Rewards, please visit',
```
Comment (jasonrsadler) [inline stories/assets/locale.ts]: nit: possessive noun `creator's`

Comment (rossmoody) [reply stories/assets/locale.ts]: great spot

**Comment by jasonrsadler** [comment]
@rossmoody please rebase when you can.

**Pull request merged**